features/aanschrijfwijze.feature

```diff
@@ -508,7 +508,7 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
       | fields              | naam.aanschrijfwijze            |
       Dan bevat de persoon met burgerservicenummer '999992934' de volgende 'naam' gegevens
       | naam                  | waarde            |
-      | gebruikinlopendetekst | <aanschrijfwijze> |
+      | aanschrijfwijze.naam  | <naam in aanschrijfwijze> |
 
       Voorbeelden:
       | geslacht | geslachtsnaam     | naamgebruik | geslacht partner | aanschrijfwijze              |
```

Comment:
dan moet dat hier ook worden aangepast naar "naam in aanschrijfwijze":
| geslacht | geslachtsnaam | naamgebruik | geslacht partner | naam in aanschrijfwijze |

Reply:
Klopt!

Reply:
Aangepast